ts/screens/profile/ProfileDataScreen.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import { List } from "native-base";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import I18n from "../../i18n";
+import { GlobalState } from "../../store/reducers/types";
+import {
+  profileEmailSelector,
+  isProfileEmailValidatedSelector,
+  hasProfileEmailSelector
+} from "../../store/reducers/profile";
+import TopScreenComponent from "../../components/screens/TopScreenComponent";
+import { ContextualHelpPropsMarkdown } from "../../components/screens/BaseScreenComponent";
+import ScreenContent from "../../components/screens/ScreenContent";
+import ListItemComponent from "../../components/screens/ListItemComponent";
+import { EdgeBorderComponent } from "../../components/screens/EdgeBorderComponent";
+import {
+  navigateToEmailInsertScreen,
+  navigateToEmailReadScreen
+} from "../../store/actions/navigation";
+
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "profile.preferences.contextualHelpTitle",
+  body: "profile.preferences.contextualHelpContent"
+};
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const ProfileDataScreen: React.FC<Props> = ({
+  optionEmail,
+  isEmailValidated,
+  navigateToEmailReadScreen,
+  navigateToEmailInsertScreen,
+  hasProfileEmail
+}): JSX.Element => {
+  const onPressEmail = () => {
+    if (hasProfileEmail) {
+      navigateToEmailReadScreen();
+      return;
+    }
+    navigateToEmailInsertScreen();
+  };
+
+  return (
+    <TopScreenComponent
+      contextualHelpMarkdown={contextualHelpMarkdown}
+      faqCategories={["profile", "privacy", "authentication_SPID"]}
+      goBack
+    >
+      <ScreenContent
```

Comment:
There is a reason why do you use this component instead standard typography components? It seems an old component not much used in the app.

Reply:
> There is a reason why do you use this component instead standard typography components? It seems an old component not much used in the app.

@debiff no, there is no reason. I simply used the same structure as the other screens inside the profile section. If there is a more specific component to use please tell me so I will replace it.

Reply:
@debiff maybe it's BaseScreenComponent?